prosemirror-lwdita/src/commands.ts

```diff
@@ -20,8 +19,6 @@ export function createNode(type: NodeType, args: Record<string, any> = {}): Node
   switch (type.name) {
     case 'p': return type.createAndFill() as Node;
     case 'data': return type.createAndFill({}, type.schema.text('text')) as Node;
-    case 'stentry': return type.createAndFill({}, createNode(type.schema.nodes['p'])) as Node;
```

Comment:
Why have these been removed?

Reply:
it's duplicated we had 2 entries for  `stentry` and `strow`